packages/cadl-ranch-specs/http/models/dictionary/main.cadl

```diff
@@ -0,0 +1,53 @@
+import "@azure-tools/cadl-azure-core";
+import "@azure-tools/cadl-ranch-expect";
+
+using Cadl.Http;
+using Azure.Core.Foundations;
+
+@serviceTitle("Dictionary in Complex Test Service")
```

Comment:
For cadl-ranch we want to be sure that every scenario is valid in itself. You can use `body complex` as inspiration to find missing scenarios in the testserver, but the test shouldn't be tied to body complex in any way